iron-menu-behavior.html

```diff
@@ -148,7 +148,12 @@
         var item = this.items[(curFocusIndex - i + length) % length];
         if (!item.hasAttribute('disabled')) {
           this._setFocusedItem(item);
-          return;
+
+          // Focus might not have worked, if the element was hidden or not
+          // focusable. In that case, try again.
+          if (document.activeElement == item) {
```

Comment:
@danbeam yeah, but in your example you're focusing `<x-foo>`, because that's the item distributed in the menu. I don't think I understand your example